src/trans/gpu/internal/trgtol_mod.F90

```diff
@@ -120,7 +120,7 @@ SUBROUTINE TRGTOL(ALLOCATOR,HTRGTOL,PREEL_REAL,KF_FS,KF_GP,KF_UV_G,KF_SCALARS_G,
     USE BUFFERED_ALLOCATOR_MOD, ONLY: BUFFERED_ALLOCATOR, ASSIGN_PTR, GET_ALLOCATION
     USE OPENACC_EXT,            ONLY: EXT_ACC_ARR_DESC, EXT_ACC_PASS, EXT_ACC_CREATE, &
       &                               EXT_ACC_DELETE
-    USE OPENACC,                ONLY: ACC_HANDLE_KIND
+    USE OPENACC,                ONLY: ACC_IS_PRESENT
```

Comment:
Should we still be importing `ACC_HANDLE_KIND`? It's referenced further down.

Reply:
I think so, yes. Fixing...